frontend/packages/dev-console/src/components/topology/componentFactory.ts

```diff
@@ -194,6 +197,30 @@ class ComponentFactory {
                   ),
                 ),
               );
+            case ModelKind.node:
+              return withCreateConnector(createConnectorCallback(this.hasServiceBinding))(
+                withDndDrop<
+                  any,
+                  any,
+                  { droppable?: boolean; hover?: boolean; canDrop?: boolean },
+                  NodeProps
+                >(nodeDropTargetSpec)(
+                  withEditReviewAccess('patch')(
+                    withDragNode(nodeDragSourceSpec(type))(
+                      withSelection(
+                        false,
+                        true,
+                      )(
+                        withContextMenu(
+                          workloadContextMenu,
+                          document.getElementById('modal-container'),
+                          'odc-topology-context-menu',
+                        )(WorkloadNode),
+                      ),
+                    ),
+                  ),
+                ),
+              );
```

Comment:
Why is this here? We should already have explicit support for each known type already.

Reply:
It was overlooked while cleanup.
removed it.

Reply:
@nemesis09  please remove this block.

Reply:
@nemesis09  Please remove this code